hicommonwealth/commonwealth
# [ERC20 Launchpad] - Endpoint integration for Community information step

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/9208

## Description of Changes
Added endpoint integration for Community information step

## "How We Fixed It"
N/A

## Test Plan
- Enable `FLAG_TOKENIZED_COMMUNITY`
- Visit `/createTokenCommunity`
- Open `/packages/commonwealth/client/scripts/views/pages/LaunchToken/useCreateCommunity.ts` and add your currently logged-in Ethereum wallet address for the `setSelectedAddress` state - this is only temporary for testing - the actual address selection happens in step 1 of this flow
- Verify the form is prefilled with data constructed from sample token information
- Submit the form and verify that a new token community has been created.

## Deployment Plan
N/A

## Other Considerations
N/A
